package.json

```diff
@@ -13,21 +13,21 @@
     "test": "./node_modules/.bin/eslint components/*.js pages/*.js"
   },
   "dependencies": {
-    "axios": "^0.16.0",
+    "axios": "^0.16.2",
     "moment": "^2.18.1",
-    "next": "^2.0.1",
-    "prop-types": "^15.5.7",
-    "react": "^15.4.2",
+    "next": "^2.4.9",
+    "prop-types": "^15.5.10",
+    "react": "^15.6.1",
     "react-dom": "^15.4.2",
     "shortid": "^2.2.8"
   },
   "devDependencies": {
-    "eslint": "^3.19.0",
-    "eslint-loader": "^1.7.1",
-    "eslint-config-standard": "^10.2.0",
-    "eslint-plugin-react": "^6.10.3",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-node": "^4.2.2",
+    "eslint": "^4.3.0",
+    "eslint-loader": "^1.9.0",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-react": "^7.1.0",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-node": "^5.5.1",
```

Comment:
```shell
detroitjs-com ▶︎ imcodingideas-build-warning ✓
⚡  npm i
npm ERR! code ETARGET
npm ERR! notarget No matching version found for eslint-plugin-node@^5.5.1
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'detroitjs.com'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tjmiller/.npm/_logs/2017-07-29T13_56_57_593Z-debug.log
```

With `npm install` I get the following error. With yarn it give me a prompt to select the correct dependency.